src/DotNet.Status.Web/Models/IssueMentionForwarder.cs

```diff
@@ -0,0 +1,97 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using DotNet.Status.Web.TeamsMessages;
+using JetBrains.Annotations;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+
+namespace DotNet.Status.Web.Models
+{
+    public class IssueMentionForwardingOptions
+    {
+        public string[] WatchedTeams { get; set; }
+        public string[] IgnoreRepos { get; set; }
+        public string TeamsWebHookUri { get; set; }
+    }
+
+    public interface IIssueMentionForwarder
+    {
+        Task<bool> HandleIssueBody([CanBeNull] string oldBody, string newBody, string title, string commentUri, string username, DateTimeOffset date);
+    }
+
+    public class IssueMentionForwarder : IIssueMentionForwarder
+    {
+        private readonly IOptionsSnapshot<IssueMentionForwardingOptions> _options;
+        private readonly ILogger<IssueMentionForwarder> _logger;
+        private readonly IHttpClientFactory _httpClientFactory;
+
+        public IssueMentionForwarder(IHttpClientFactory httpClientFactory, IOptionsSnapshot<IssueMentionForwardingOptions> options, ILogger<IssueMentionForwarder> logger)
+        {
+            _httpClientFactory = httpClientFactory;
+            _options = options;
+            _logger = logger;
+        }
+
+        public async Task<bool> HandleIssueBody([CanBeNull] string oldBody, string newBody, string title, string commentUri, string username, DateTimeOffset date)
+        {
+            var teams = _options.Value.WatchedTeams;
+            var foundTeam = teams.FirstOrDefault(newBody.Contains);
+            if (string.IsNullOrEmpty(foundTeam))
+            {
+                return false;
+            }
+            if (!string.IsNullOrEmpty(oldBody))
+            {
+                var oldTeam = teams.FirstOrDefault(oldBody.Contains);
```

Comment:
The first is a bit... odd... :-) If we have more than one "teams", this code is going to behave oddly.
* `(at)a` -> `(at)b` will trigger a notification
* `(at)a` -> `(at)a (at)b` will not
* `(at)a` -> `(at)b (at)a` will

This pattern doesn't make a whole lot of sense... it seems like it only REALLY works if there is exactly one team we are following... so let's just only support following one team.

Either that or this code needs to get a lot more complicated. :-(

Reply:
Yea, one team works.